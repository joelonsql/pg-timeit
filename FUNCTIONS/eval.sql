CREATE OR REPLACE FUNCTION timeit.eval
(
    internal_function text,
    input_values text[]
)
RETURNS text
LANGUAGE c
AS '$libdir/timeit', 'eval';
