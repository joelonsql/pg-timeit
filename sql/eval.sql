SELECT timeit.eval('numeric_add', ARRAY['1','2']);
