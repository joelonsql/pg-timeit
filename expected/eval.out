SELECT timeit.eval('numeric_add', ARRAY['1','2']);
 eval 
------
 3
(1 row)

